client/src/components/SignIn/index.js

```diff
@@ -59,13 +59,15 @@ class SignInFormBase extends Component {
         <input
           name="email"
           value={email}
+          autoComplete="username"
```

Comment:
What is the purpose of autocomplete? Doesn't the webpage already do this?

Reply:
The most recent version of Chrome does, but for increased accessibility across different platforms, the autoComplete attribute helps browsers identify what types they are (it was just a console warning I got while running React so I fixed it, but  [https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete](here's more info))

Reply:
Interesting, good to know!